lib/Controller/PublicViewController.php

```diff
@@ -97,15 +97,21 @@ public function publicIndexForEmbedding(string $token):TemplateResponse {
 	 */
 	private function publicIndex(string $token,
 								 string $renderAs):TemplateResponse {
+		$defaultInitialView = $this->config->getAppValue($this->appName, 'currentView', 'dayGridMonth');
+		$defaultShowWeekends = $this->config->getAppValue($this->appName, 'showWeekends', 'yes');
+		$defaultWeekNumbers = $this->config->getAppValue($this->appName, 'showWeekNr', 'no');
+		$defaultSkipPopover = $this->config->getAppValue($this->appName, 'skipPopover', 'no');
```

Comment:
We had `'skip_popover' => true` as default previously, but this is fine.

Reply:
Yeah, let me change that again.